blockchain/contracts/FlatNFTtheWorld.sol

```diff
@@ -1262,7 +1264,7 @@ contract NFTtheWorld {
 
     mapping(address => mapping(uint256 => string[]))
         private nftReservationInformationOfUsers;
-    mapping(address => uint256[]) public nftAssetsInformationOfUsers;
+    mapping(address => uint256[]) private nftAssetsInformationOfUsers;
```

Comment:
If this is not needed anymore @steinkohl, then we could get rid of it. What do you think? We are getting this information directly from the wallet now, right?

Reply:
Yes, I overlooked that. Since we can fetch the owner directly from the NFTs, this is no longer necessary.